tests/SBRPTests/CommandHelper.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace SBRPTests;
+internal class CommandHelper
```

Comment:
Consider coping https://github.com/dotnet/installer/blob/main/src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ExecuteHelper.cs.  One nice thing about it is that is logs what commands are being executed as well as their output.  This is useful when a test fails in that you can look at the output to help diagnose the issue without having to reproduce the issue locally.

Reply:
Got it. Will update with this.